ansible/roles/iroha-docker/tasks/init-vars.yml

```diff
@@ -27,9 +27,15 @@
       iroha_peers_map: "{{ iroha_peers_map }} + [ {{ {'hostname': item, 'human_hostname': 'c_' + item | regex_replace('\\.', '_') | regex_replace(':', '_'), 'peer_port': item.split(':')[1] } }} ]"
     loop: "{{ iroha_nodes }}"
 
+  - set_fact:
```

Comment:
I notice we have a lot of `set_fact` task , we can combine it like this: 
```
    - set_fact:
        a: "aaaaaaaaa"
        b: "bbbbbbbbb"

    - debug:
        var: a, b

```

Reply:
I don't think we can do it. There are different preconditions when to run certain `set_fact` task. Also, it would become hard to trace the role execution step-by-step if a particular task becomes larger.